Yoast/javascript
# Use Webpack to load css

Fixes https://github.com/Yoast/javascript/issues/160

## Summary 
We tried to load plugin.css from the node_modules with a static path reference.
Because of lerna+yarn+workspaces and their hoisting, we don't know the exact path in every situation, so we have to use Webpack imports to resolve the location of the file.

## Relevant technical choices:
As described by https://jestjs.io/docs/en/webpack.html, I now mock all imported .css files. This is because we now load the draft-js plugin style trough webpack instead of through an HTML import.

## Test instructions
This PR can be tested by following these steps:
* yarn install and yarn start in `javascript/apps/components`
* Go to the snippet preview tab
* Click Edit snippet
* See that the replace variables have purple styling and there is the right amount of padding to it. (see screenshot)

**new:**
![image](https://user-images.githubusercontent.com/5352634/55235662-5fc17d00-522e-11e9-9106-71f5644781e7.png)
**old:**
![image](https://user-images.githubusercontent.com/5352634/55235763-94353900-522e-11e9-833f-2f4b5c85c8ed.png)


## UI changes

## Quality assurance


Fixes #
